src/main/webapp/app/modules/canvas/filter/filter-element.tsx

```diff
@@ -66,7 +66,7 @@ const FilterElement = (props: IFilterElementProp) => {
       dataType: props.feature.type,
       valueType: 'valueType',
```

Comment:
as mentioned over chat remove filters code for now..keep it in your local until filters changes are done